Bottom: 364948e8175b94c5eb89a88ee72798e461ce8a50
Top:    ab82ac2be62b27234324b374b250afe4f2656272
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-21 15:13:16 +0000

More debug stuff


---

diff --git a/src/polygon1.c b/src/polygon1.c
index c86b382..806126b 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2956,6 +2956,8 @@ M_Set_Parentage (POLYAREA *poly, POLYPARENTAGE parentage)
 
 }
 
+static void clear_marks (POLYAREA * p);
+
 /* just like poly_Boolean but frees the input polys */
 int
 poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
@@ -2967,13 +2969,12 @@ poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
   int code;
   CVCList *the_list;
   POLYAREA *a_copy, *b_copy;
+  CVCList *the_list;
 
   *res = NULL;
 
   g_warning ("BEGIN BOOLEAN");
 
-  *res = NULL;
-
   /* Make copies for tracking polygon parentage (DEBUG) */
   if (!poly_M_Copy0 (&a_copy, a) || !poly_M_Copy0 (&b_copy, b))
       return err_no_memory;
@@ -2990,7 +2991,6 @@ poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
       M_Set_Parentage (b_copy, b->parentage);
       M_Set_Parentage (b, no_parentage);
     }
->>>>>>> patched
 
   if (!a)
     {
